tests/e2e/steps_consumer_misbehaviour.go

```diff
@@ -0,0 +1,263 @@
+				// of ICS before the soft opt-out threshold was introduced, we need to set the
+				// soft opt-out threshold to 0.05 in the consumer genesis to ensure that the
+				// consumer binary doesn't panic. Sdk requires that all params are set to valid
+				// values from the genesis file.
+				genesisChanges: ".app_state.ccvconsumer.params.soft_opt_out_threshold = \"0.05\"",
+			},
+			state: State{
+				chainID("provi"): ChainState{
+					ValBalances: &map[validatorID]uint{
+						validatorID("alice"): 9500000000,
+						validatorID("bob"):   9980000000,
+					},
+				},
+				chainID(consumerName): ChainState{
+					ValBalances: &map[validatorID]uint{
+						validatorID("alice"): 10000000000,
+					},
+				},
+			},
+		},
+		{
+			action: addIbcConnectionAction{
+				chainA:  chainID(consumerName),
+				chainB:  chainID("provi"),
+				clientA: 0,
+				clientB: 0,
+			},
+			state: State{},
+		},
+		{
+			action: addIbcChannelAction{
+				chainA:      chainID(consumerName),
+				chainB:      chainID("provi"),
+				connectionA: 0,
+				portA:       "consumer", // TODO: check port mapping
+				portB:       "provider",
+				order:       "ordered",
+			},
+			state: State{},
+		},
+		// delegate some token and relay the resulting VSC packets
+		// in oder to initiates the CCV channel
+		{
+			action: delegateTokensAction{
+				chain:  chainID("provi"),
+				from:   validatorID("alice"),
+				to:     validatorID("alice"),
+				amount: 11000000,
+			},
+			state: State{
+				chainID("provi"): ChainState{
+					ValPowers: &map[validatorID]uint{
+						validatorID("alice"): 511,
+						validatorID("bob"):   20,
+					},
+				},
+				chainID(consumerName): ChainState{
+					ValPowers: &map[validatorID]uint{
+						validatorID("alice"): 500,
+						validatorID("bob"):   20,
+					},
+				},
+			},
+		},
+		{
+			action: relayPacketsAction{
+				chainA:  chainID("provi"),
+				chainB:  chainID(consumerName),
+				port:    "provider",
+				channel: 0,
+			},
+			state: State{
+				chainID(consumerName): ChainState{
+					ValPowers: &map[validatorID]uint{
+						validatorID("alice"): 511,
+						validatorID("bob"):   20,
+					},
+				},
+			},
+		},
+	}
+
+	return s
+}
+
+// stepsCauseConsumerMisbehaviour forks a consumer chain in order to cause a ICS misbehaviour.
+// The malicious validator behind the attack gets jailed and the consumer client freezed on the provider.
+func stepsCauseConsumerMisbehaviour(consumerName string) []Step {
+	consumerClientID := "07-tendermint-0"
+	forkRelayerConfig := "/root/.hermes/config_fork.toml"
+	return []Step{
+		{
+			// start a consumer chain's validator clone
+			action: forkConsumerChainAction{
+				consumerChain: chainID(consumerName),
+				providerChain: chainID("provi"),
+				validator:     validatorID("alice"),
+				relayerConfig: forkRelayerConfig,
+			},
+			state: State{},
+		},
+		{
+			// start relayer to detect ICS misbehaviour
+			action: startRelayerAction{},
```

Comment:
The relayer is initially connected to the `provider` and the "original" `consumer` that has not forked?

Maybe write that as a comment.